tox.ini

```diff
@@ -38,7 +38,7 @@ deps =
     docker-compose=={[tox]docker_compose_version}
 commands =
     docker-compose down
-    docker-compose --verbose build
+    docker-compose --verbose build --build-arg PIP_INDEX_URL={env:PIP_INDEX_URL:https://pypi.yelpcorp.com/simple}
```

Comment:
I think possibly you are not seeing a line about `pypi.python.org` in your fluffy about because that layer was cached:
```
Step 9/14 : RUN pip install -r requirements.txt
 ---> Using cache
 ---> a3db76afc0e7
```

Reply:
Good catch!
I updated missed Dockerfiles, will add fresh logs after running tests.